andela/prometheus-ah
# #159987746: implement get all user profile

#### What does this PR do?
- Get the lists of all users for authenticated user
#### Description of Task to be completed?
- Write tests to for all edges in the controller
- Write code to make the test pass
- Update documentation
#### How should this be manually tested?
- clone this repository
- run npm install
- navigate to this branch by using ```git checkout ft-get-all-users-159987746```
- check ```.env.example``` to set up your ```.env``` file
- on your terminal type,```npm run dev``` to start the server
- Make sure your Postgres database is running
- Launch postman, insert the authorization token in the header and make a GET request to ```localhost:port/api/profiles```
#### What are the relevant pivotal tracker stories?
- [#159987746](https://www.pivotaltracker.com/n/projects/2192625/stories/159987746/comments/193942520)
#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-09-06 at 8 07 19 pm" src="https://user-images.githubusercontent.com/23406358/45179819-83c95c80-b211-11e8-98f1-c86e760b4ebe.png">
